server/src/main/java/io/spine/server/entity/storage/Column.java

```diff
@@ -40,9 +42,12 @@
  * <p>If there are repeated column names within an {@code Entity},
  * the exception will be raised when a repository serving the entity is added to
  * its {@code BoundedContext}.
+ *
+ * <p>Please note this feature is experimental.
```

Comment:
How come? It's been here for years. What is it in entity columns that we're not sure about?

Reply:
Please see #1135.